spec/lib/search_spec.rb

```diff
@@ -115,6 +115,37 @@
     end
   end
 
+  context "with apostrophes" do
+    fab!(:post_1) { Fabricate(:post, raw: "searching for: John's") }
+    fab!(:post_2) { Fabricate(:post, raw: "searching for: Johns") }
+
+    before do
+      SearchIndexer.enable
+    end
+
+    after do
+      SearchIndexer.disable
+    end
+
+    it "returns correct results" do
+      SiteSetting.search_ignore_accents = false
```

Comment:
It looks like both tests return the same results no matter the value of the `search_ignore_accents` site setting.

Do we really need that check? 🤔